newrelic/structures_newrelic_synthetics_script_browser_monitor.go

```diff
@@ -104,6 +105,9 @@ func buildSyntheticsScriptBrowserUpdateInput(d *schema.ResourceData) synthetics.
 		if runtimeTypeVersionOk {
 			input.Runtime.RuntimeTypeVersion = synthetics.SemVer(runtimeTypeVersion.(string))
 		}
+	} else {
+		input.Runtime.RuntimeType = ""
+		input.Runtime.RuntimeTypeVersion = ""
```

Comment:
Test - with a Scripted Browser monitor already created, it can be moved from the new runtime to the legacy runtime and vice versa.

https://github.com/user-attachments/assets/f5e083cb-d3f0-436d-9d85-d1242157e1b6